work-item-comments.go

```diff
@@ -121,7 +128,7 @@ func WorkItemIncludeCommentsAndTotal(ctx context.Context, db application.DB, par
 	go func() {
 		defer close(count)
 		application.Transactional(db, func(appl application.Application) error {
-			cs, err := appl.Comments().List(ctx, parentID)
+			cs, err := appl.Comments().ListAll(ctx, parentID)
```

Comment:
Move this over to a Count function or something that runs a count sql. We don't need to load the whole data set and count the objects.